src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -40,23 +51,49 @@ public class PersonCard extends UiPart<Region> {
     private Label moneyOwed;
     @FXML
     private FlowPane daysAvailable;
+    private TranslateTransition moveTransition;
+    private TranslateTransition moveBackTransition;
+    private FadeTransition fadeInTransition;
 
     /**
      * Creates a {@code PersonCode} with the given {@code Person} and index to display.
      */
-    public PersonCard(Person person, int displayedIndex) {
+    public PersonCard(Person person, int displayedIndex, boolean animate) {
```

Comment:
Ooops sorry for the minor nit but `animate` is a boolean variable so should be named sth like `hasAnimation` or `isAnimationEnabled`